spi_nand_flash/test_app/main/test_app_main.c

```diff
@@ -18,7 +18,7 @@ void setUp(void)
 void tearDown(void)
 {
     esp_reent_cleanup();    //clean up some of the newlib's lazy allocations
-    unity_utils_evaluate_leaks_direct(20);
+    unity_utils_evaluate_leaks_direct(200);
```

Comment:
Do you know what part of this change requires higher leak threshold?

Reply:
Honestly, I’m not entirely sure which change is causing this leak, and I couldn’t pinpoint it exactly. It’s an 88-byte leak, actually. I will change the threshold to 100. Need to recheck to find the cause of this leak.

Reply:
Got it. The memory leak was in dhara_glue.c. Memory was allocated to spi_nand_flash_dhara_priv_data_t but was not freed.